rest/komand_rest/util/util.py

```diff
@@ -9,9 +13,45 @@ def merge_dicts(x, y):
         return z
 
     '''Copy the case insensitive headers dict to a normal one'''
+
     @staticmethod
     def copy_dict(x):
         d = {}
         for key in x:
             d[key] = x[key]
         return d
+
+    @staticmethod
+    def call_api(url, path, headers, ssl_verify, auth=None):
```

Comment:
This function should raise PluginExceptions. The connection test should catch PluginExceptions and re-raise them as CTE's.

Reply:
This function is used only in connection. So for this moment there is no purpouse to raise PluginException. 
But if you want this change, we can change it here :)